contrib/localnet/scripts/start-zetacored.sh

```diff
@@ -219,11 +219,11 @@ then
   fi
 
 # set admin account
-  zetacored add-genesis-account zeta1srsq755t654agc0grpxj4y3w0znktrpr9tcdgk 100000000000000000000000000azeta
+  zetacored add-genesis-account zeta1svzuz982w09vf2y08xsh8qplj36phyz466krj3 100000000000000000000000000azeta
```

Comment:
Lets define a variable fo this address

Reply:
Yeah I will do that in my upcoming PRs based on https://github.com/zeta-chain/node/pull/2308